docs/book/customer_and_shopuser.rst

```diff
@@ -0,0 +1,83 @@
+.. index::
+   single: Customer and ShopUser
+
+Customer and ShopUser
+=====================
+
+For handling customers of your system **Sylius** is using a combination of two entities - **Customer** and **ShopUser**.
+The difference between these two entities is simple:
+the **Customer** is unregistered and the **ShopUser** is registered in the system - they have an account.
```

Comment:
`the **Customer** is a guest in our shop and the **ShopUser** is a registered user in the system - he has an account.`